proposal.Rmd

```diff
@@ -0,0 +1,22 @@
+---
+title: "Proposal: shades of foundation offered by a variety of beauty brands around the world"
+output: github_document
+---
+
+# Section 1: Motivation and Purpose
+
+**Our role**: data scientist consultancy firm
+
+**Target audience**: 
+
+- Women who are interested in makeup shades of foundation offered by beauty brands; or
```

Comment:
To me makeup foundation sounds a bit funny but I think its fine to leave as is. I think simply foundation will do but would love everyone else's opinion on this.

Reply:
sounds good on this point!